### PR TITLE
Refactor: Add Init method to the class

### PR DESCRIPTION
**Pull request by keyur9**
Add: Support to fetch user Id from authorization header
Add: Helper functions to create event model
Add: Helper functions for config options
Refactor: Add Init method to the class
Refactor: Add Classmethod to the function who doesn't need class instance
Refactor: Test for Django1.9 and above & support for Python 2 and 3
Refactor: Update README.md
Bump version to 1.7.6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,12 @@ to add custom metadata that will be associated with the event. The metadata must
 #### __`MASK_EVENT_MODEL`__
 (optional) _(EventModel) => EventModel_, a function that takes an EventModel and returns an EventModel with desired data removed. Use this if you prefer to write your own mask function than use the string based filter options: REQUEST_BODY_MASKS, REQUEST_HEADER_MASKS, RESPONSE_BODY_MASKS, & RESPONSE_HEADER_MASKS. The return value must be a valid EventModel required by Moesif data ingestion API. For details regarding EventModel please see the [Moesif Python API Documentation](https://www.moesif.com/docs/api?python).
 
+#### __`AUTHORIZATION_HEADER_NAME`__
+(optional) _string_, A request header field name used to identify the User in Moesif. Default value is `authorization`.
+
+#### __`AUTHORIZATION_USER_ID_FIELD`__
+(optional) _string_, A field name use to parse the User from authorization header in Moesif. Default value is `sub`.
```
Comment (dkm199) [inline README.md:125]: same comment

File: moesifdjango/logger_helper.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            # Fix the padding issue before decoding
+            token += '=' * (-len(token) % 4)
+            # Decode the payload
+            base64_decode = base64.b64decode(token)
+            # Transform token to string to be compatible with Python 2 and 3
+            base64_decode = self.transform_token(base64_decode)
+            # Convert the payload to json
+            json_decode = json.loads(base64_decode)
+            # Check if field is present in the body
+            if field in json_decode:
```
Comment (dkm199) [inline moesifdjango/logger_helper.py:145]: similar comment, may be makes sense to add lower case support?

**Review by dkm199 — APPROVED** [review]
lgtm, minor comments

File: moesifdjango/logger_helper.py (hunk 2 above)
Comment (keyur9) [reply moesifdjango/logger_helper.py:147]: Have added the support and updated the PR.

**Review by dgilling — APPROVED** [review]

**Pull request merged**